templates/collection_view_screen/app/screens/name_screen.rb

```diff
@@ -0,0 +1,59 @@
+class <%= @name_camel_case %>Screen < UICollectionViewController
+  include ProMotion::ScreenModule
+
+  title "Your title here"
+  stylesheet <%= @name_camel_case %>ScreenStylesheet
+
+  <%= @name.upcase %>_CELL_ID = "<%= @name_camel_case %>Cell"
+
+  def self.new(args = {})
+    # Set layout
+    layout = UICollectionViewFlowLayout.alloc.init
+    s = self.alloc.initWithCollectionViewLayout(layout)
+    s.screen_init(args) if s.respond_to?(:screen_init)
+    s
```

Comment:
We should have a general discussion about .tap. One thing it does is cause a `self` problem because of the block. I use to use tap in this situation, and now I'm rethinking that.